dts/bindings/display/nxp,dcnano-lcdif.yaml

```diff
@@ -32,3 +32,11 @@ properties:
       - "24-bit" # 24 Bit
     description:
       LCD data bus width. The default is set to the reset value of 24-bit
+
+  version:
```

Comment:
Not opposed to this approach, but is this an IP _revision_, or a new IP block? If it is a new IP block, we can actually support multiple compatibles within one driver (and we should, in that case). See here for an example: https://github.com/zephyrproject-rtos/zephyr/blob/e60f04096cd8d9227c8279ca830e72bc1b5b8b1c/drivers/i2c/i2c_tca954x.c#L204

Reply:
Thanks, I did not know about this usage before. I think it is an IP revision, for DC8000 there are extra registers to set for a frame configuration compared to DCNano. Actually I did not find any similar use case of adding a new binding for IP revision so I'm not sure it is good practice. I did find using kconfig symbols to isolate extra register configurations but I think it is obsolete? https://github.com/zephyrproject-rtos/zephyr/blob/e60f04096cd8d9227c8279ca830e72bc1b5b8b1c/drivers/counter/counter_mcux_snvs.c#L279